platform/android/export/export.cpp

```diff
@@ -2128,6 +2207,8 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 			}
 			cmdline.push_back("build");
 			cmdline.push_back("-Pexport_package_name=" + package_name); // argument to specify the package name.
+			cmdline.push_back("-Pversion_code=" + version_code); // argument to specify the version code.
+			cmdline.push_back("-Pversion_name=" + version_name); // argument to specify the version name.
```

Comment:
Can you prefix it with `export` to remain consistent (e.g: `export_version_code`, `export_version_name`).